### PR TITLE
feat: Landing page added.

### PR DESCRIPTION
**Pull request by rutu14**
Landing Page added.
And Router routes and Links too.

[✔ Please check these files ](https://github.com/rutu14/Ecommerce-React/pull/3/files/c1d02321badc1e0712ce25e2e4b4730207ca9ea0..5d5a6be268cc82d480a60b4b325c838e6ed45e02?file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/components/CategorySection/CategorySection.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <>
+        <h3 id="category" className="heading2 medium text-center m20">Category</h3>
+        { loader && <Loader/>}
+        <section className="grid-3-cols below-hero">
+            {categoryData && categoryData.map((item) => (
+                <Link to={`product/${item.categoryName}`} className="card card-box-shadow cp category-card" key={item._id}>
+                    <div className="card-title-box middle-overlay title-color">
+                        <h2 className="heading3 text-center medium">{item.categoryName}</h2>
+                    </div>
+                    <img className="card-img card-img-all-borders category-card-img" src={item.image}/>
```
Comment (santhoshpemmaka) [inline src/components/CategorySection/CategorySection.jsx:35]: If possible better use alt="" statement inside image tag.

File: src/pages/Product-Page/ProductPage.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+const ProductPage = () => {
+
```
Comment (santhoshpemmaka) [inline src/pages/Product-Page/ProductPage.jsx:3]: Remove extra lines inside the component file.

**Review by santhoshpemmaka — APPROVED** [review]

**Comment by santhoshpemmaka** [comment]
Code looking good 💯 . Good to merge 👍

File: src/components/CategorySection/CategorySection.jsx (hunk 1 above)
Comment (rutu14) [reply src/components/CategorySection/CategorySection.jsx:35]: Thanks for the reminder.

**Review by srishtigautam21 — APPROVED** [review]
The size of PR is small
Adding that link in description made the PR review easy
The code is well understandable
Good to go

**Pull request merged**